cmd/getSchema.go

```diff
@@ -15,18 +16,30 @@ var getSchemaCmd = &cobra.Command{
 	Use:   "schema [table]",
 	Short: "Get table schema",
 	Long:  `Get schema from table name`,
+	Args:  cobra.ExactArgs(1),
 	Run:   getSchema,
 }
 
 func getSchema(cmd *cobra.Command, args []string) {
-	fmt.Print("Hello")
-
-	dbParams := pkg.ConnectParams{
-		Host:     "localhost",
-		Port:     5321,
-		User:     "admin",
-		Password: "admin",
-		Dbname:   "test",
+	tableName := args[0]
```

Comment:
I think we can actually make the query get everything from all of the tables in the db so instead of defining the table name, we can just get everything from all the tables. but I could misunderstand how you're planning to use this